### PR TITLE
Reconfigure Restyled

### PR DESCRIPTION
**Pull request by lucydodo**
This PR reconfigures Restyled, which was currently used as a linter.
For consistency with the original repo, I have referenced the configruation files from the original repo: https://github.com/getredash/redash/blob/master/.restyled.yaml

Feel free to leave any comments or reviews. Thank you.

### CODE REVIEW

**Comment by lucydodo** [comment]
FYI: When this PR was merged, I closed the following PRs in advance to avoid conflicts with existing configuration.
- #478
- #480
- #481
- #515
- #582
- #595
- #602
- #604
- #607
- #609
- #639
- #659

**Comment by justinclift** [comment]
@eradman Do you know the linter syntax well enough to case your eye over this? :smile:

File: .restyled.yaml
Comment (eradman) [inline .restyled.yaml:57]: Did you mean `prettier`? I'm not sure what `prettier-with-tailwindcss` is

Comment (lucydodo) [reply .restyled.yaml:57]: I referenced the official documentation, and it appears that at some point they switched to using that binary instead.
Related: https://github.com/restyled-io/restylers/commit/6c1ddee30ce8e005428244f44c23c6b255485b31

Comment (lucydodo) [reply .restyled.yaml:57]: By the way, I misunderstood your comment, if you want to run it manually,
you can install that package with the following command:
```
yarn add --dev prettier prettier-plugin-tailwindcss
```

Comment (eradman) [reply .restyled.yaml:57]: So the `restyled/restyler-prettier` image is built with tailwindcss, but I don't think we care about that.

Can we try with just the command?

```
    command:
      - prettier
      - '--write'
```

The `prettier` binary is avaialble in the image

```
$ docker run -it restyled/restyler-prettier:v3.3.2-2 prettier -v
3.3.2
```

Comment (lucydodo) [reply .restyled.yaml:57]: Sure, do you want to change that?

Comment (eradman) [reply .restyled.yaml:57]: Go ahead and make that change

Comment (lucydodo) [reply .restyled.yaml]: Cool, I made changes.

**Review by eradman — APPROVED** [review]

File: .restyled.yaml
Comment (lucydodo) [reply .restyled.yaml]: Thanks for the review, approval. :)
Can I merge it? Or do I have to wait for another review?

**Comment by justinclift** [comment]
Seems to be all good + approved, so I've just hit the merge button. :smile:

**Comment by lucydodo** [comment]
Thank you both of you. :)

**Pull request merged**
